alerts/geomodel_location.py

```diff
@@ -108,6 +132,11 @@ def _process(self, cfg: config.Config, qindex: int):
 
         search = SearchQuery(**evt_cfg.search_window)
         search.add_must(QSMatch(evt_cfg.lucene_query))
+        # Ignore empty usernames
+        search.add_must_not(TermMatch(evt_cfg.username_path, ''))
+        # Ignore whitelisted usernames
+        for whitelisted_username in cfg.whitelist.users:
+            search.add_must_not(TermMatch(evt_cfg.username_path, whitelisted_username))
```

Comment:
I moved this logic here so we aren't calling `onAggregation` for usernames we know we're going to whitelist.